docs/UserGuide.md

```diff
@@ -339,3 +395,42 @@ ProductiveNUS data are saved in the hard disk automatically after any command th
 3. Close the app in your other computer.
 4. Overwrite the newly created data file with the data file from your previous computer.
 5. All your existing data has been successfully transferred!
+
+**Q**: I am not free during the time that my assignment has been scheduled. How do I reschedule?<br>
+**A**:
+You can reschedule by using the schedule command on that assignment again.
+
+1. Find out the index of the assignment that you want to reschedule.
+
+DIAGRAM
+
+2. Type the schedule command with the correct index into the command box .
+
+DIAGRAM
+
+<div markdown="span" class="alert alert-success">
+**:bulb: Tip:**
+You can copy the entire command if you intend to reschedule the assignment more than once. 
+You can paste the command into the command box after step 3 and repeat step 2-3 until you have obtain your
+ ideal schedule.
+</div>
+
+3. Press ENTER to reschedule your assignment.
+
+DIAGRAM
+
+**Q**: I double-clicked the productivenus.jar file but the app is not starting. What should I do?
+**A**: 
+1. Go to the folder where you have downloaded and saved the productivenus.jar file.
+2. Copy the file directory of the folder. The file directory is labelled in the diagram below.
+DIAGRAM
+3. Open your command interpreter. To do so, press the Windows key, type in `cmd` and press ENTER.
+4. Your command interpreter will start, and the command interpreter should look something similar to the diagram below.
```

Comment:
It may be good to include how to open the command interpreter for Mac and Linux OS as well :)

Reply:
uhh I've no idea how to do that for Linux and Mac 🤣 @minhhhnguyen2000 @andreatanky anyone knows?

Reply:
for mac its jus called terminal LOL